utils/i2c-tools/Makefile

```diff
@@ -53,6 +55,7 @@ define Package/python-smbus
   CATEGORY:=Languages
   TITLE:=Python bindings for the SMBUS
   DEPENDS:=+libi2c +python-light
```

Comment:
if updating, it is a good idea to also change this DEPENDS to
`DEPENDS:=+PACKAGE_python-smbus:libi2c +PACKAGE_python-smbus:python-light`
similarly for `python3-smbus` it's a good idea to change to
`DEPENDS:=+PACKAGE_python3-smbus:libi2c +PACKAGE_python3-smbus:python-light`

Reply:
Well in general I think that it is better to fix broken build system than doing this. If I understand this correctly this masks this dependency because otherwise it would be build if any package from this Makefile is used as a dependency not exactly this package.. I added it but I am not pleased with this.

Reply:
you can have 2 patches in the same PR;
the first that only fixes stuff, and the 2nd that changes/enhances stuff;

to be fair, you are also doing some enhancements already with your patch, so why not go all the way with this ? :)